README.md

```diff
@@ -121,6 +121,18 @@ await invoice.isPaid(); // check the payment status as descibed above
 ### Fiat conversions
 Helpers to convert sats values to fiat and fiat values to sats.
 
+### Lightning Address Proxy
+alby-tools uses a [proxy](https://github.com/getAlby/lightning-address-details-proxy) to simplify requests to lightning providers.
+
+- Many ln addresses don't support CORS, which means fetching the data directly in a browser environment will not always work.
+- Two requests are required to retrieve lnurlp and keysend data for a lightning address. The proxy will do these for you with a single request.
+
+You can disable the proxy by explicitly setting the proxy to undefined when initializing a lightning address:
```

Comment:
@im-adithya currently it will use the default proxy
```
this.options = { proxy: DEFAULT_PROXY };
this.options = Object.assign(this.options, options);
```

Reply:
Ah, forgot about it! Sorry for the confusion!